cmd/node/main.go

```diff
@@ -567,10 +567,18 @@ func startNode(ctx *cli.Context, log *logger.Logger, version string) error {
 		"GenesisTimeStamp", startTime.Unix(),
 	)
 
+	sessionInfoFileOutput += "\nStarted with parameters:\n"
```

Comment:
You can also use here Sprintf even if there are no parameters for now

Reply:
done